### ISSUE TITLE
REST standard pressure units

### ISSUE THREAD

**Issue by nkx111** [rest-for-physics/framework#115]
I am wondering if the standard units for pressure is `atm` or `bar`. They have slight difference. In TRestDetectorGas it stores pressure in `atm`, but in TRestSystemOfUnits the units with scale 1 is `bar`. Also in TRestSystemOfUnits the conversion to torr is wrong. 760 torr = 1 atm, not 1 bar

**Comment by jgalan** [comment]
Right, I believe `bar` should be the standard unit pressure for REST. Just because we are used to measure the pressure chamber in `bar`. However, for gas calculations I believe Magboltz and Garfield use `atm`.

This should be fixed.
